addons/mail/static/src/messaging/entity/chat_window/chat_window.js

```diff
@@ -624,6 +272,13 @@ function ChatWindowFactory({ Entity }) {
                 threadInitialScrollTop,
             });
 
+            // manager
+            if (manager && this.manager !== manager) {
```

Comment:
manager should be "required" and it should crash if it is not set (should probably be a generic relation config, that should be true for causal)

Also causal should probably not be "changeable". If it is set, trying to change it should be prevented